src/command/cmd_ac.c

```diff
@@ -41,6 +41,7 @@
 #include <assert.h>
 #include <libgen.h>
 #include <dirent.h>
+#include <ctype.h>
```

Comment:
What do we need this for?

Reply:
The `isdigit` function in _vcard_autocomplete, Checks if the parameter is a number

Reply:
thanks!